app/templates/event/partials/edit_event_details_tab.html

```diff
@@ -40,15 +59,37 @@
     </select>
   </div>
 
+  <div class="form_row_end">
+      <select name="availability">
+          <option value="False">Free</option>
+          <option value="True" selected>Busy</option>
+      </select>
+      <select name="privacy">
+          <option value="private">Private</option>
+          <option value="public" selected>Public</option>
+      </select>
+  </div>
+</div>
 
-    <div class="form_row_end">
-        <select name="availability">
-            <option value="False">Free</option>
-            <option value="True" selected>Busy</option>
-        </select>
-        <select name="privacy">
-            <option value="private">Private</option>
-            <option value="public" selected>Public</option>
-        </select>
-    </div>
-</div>
\ No newline at end of file
+<script>
+  var items_num = 1;
```

Comment:
Preferably, avoid having global variables.

Reply:
Any ideas on how to store this info (number of items in the list) without using a global variable?